CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+## 1.5.0
+
+* Add 'force' command to record the unapplied migrations from the filesystem in
```

Comment:
Good point. I don't think 'fake' will work because that sounds too much like you're going to run the migration but roll it back.

What about the longer but on-the-nose 'record-only'?